ds_judgements_public_ui/templates/judgment/results_error.html

```diff
@@ -0,0 +1,22 @@
+{% extends "layouts/base.html" %}
+{% load utilities search_results_filters static %}
+{% block title %}
+  Search results - Find Case Law
```

Comment:
is this section duplictae from the actual search results template? Could we abstract it away?

Also nitpick; could we name the file `search_results_error.html` for consistency